common/student/activation.ts

```diff
@@ -90,6 +92,13 @@ export async function deactivateStudent(student: Student, silent: boolean = fals
             });
             // TODO Notify participants
         }
+        // courses[i].meetingId && await deleteZoomMeeting(courses[i].meetingId);
```

Comment:
TODO?

Reply:
the todo was added, I guess, to send notifications to pupils that still have courses with this student. the other line was a relict. I removed it.